3rdparty/compiler-rt/CMakeLists.txt

```diff
@@ -5,9 +5,9 @@ include(FetchContent)
 FetchContent_Declare(
   compiler-rt-sources
   SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/compiler-rt
-  URL https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.1/compiler-rt-10.0.1.src.tar.xz
+  URL https://github.com/llvm/llvm-project/releases/download/llvmorg-11.1.0/compiler-rt-11.1.0.src.tar.xz
```

Comment:
@CyanDevs  @anakrish  @asvrada  - Would 11.1 LLVM work with clang-10? After this is merged, we can only build main with clang-11 correct?

Reply:
According to https://github.com/openenclave/openenclave/pull/4662/files#r1021092938 this should work. I can do another test to make sure.

Reply:
OE built this successfully with only clang-10 installed